src/snappy_device_agents/devices/muxpi/muxpi.py

```diff
@@ -236,13 +236,13 @@ def get_image_type(self):
         def check_path(dir):
             self._run_control("test -e {}".format(dir))
 
-        # First check if this is a limerick image
+        # First check if this is an oem image
         try:
             disk_info_path = self.mount_point / "writable/.disk/info"
-            self._run_control(f"grep limerick {disk_info_path}")
-            return "limerick"
```

Comment:
Is there something less generic than "oem" we can use to describe it? Is there a more general term for just limerick and erlangen? I'm concerned that this could get confused with other oem image types that might not work using this method.

Reply:
Thank you, do you think "ce-jenkins" will be better as all those images are built by CE Jenkins and they're using same build script, which means the same build stamp rule applies to both of them.

Reply:
ce-jenkins is just where you kick off the builds though right? I don't think it has anything to do with the "type" of images that these are. What is it that makes these (and presumably future images) the same, so that they can be used the same way?